manifest.json

```
@@ -1,6 +1,7 @@
 {
     "id": "readwise-mirror",
     "name": "Readwise Mirror",
+    "version": "2.4.1-beta.1",
```

Biome (2.5.0)

[error] The key version was already declared.

(lint/suspicious/noDuplicateObjectKeys)